apps/examples/e2e/tests/test-ui.spec.ts

```diff
@@ -1,10 +1,109 @@
-import test from '@playwright/test'
+import test, { expect } from '@playwright/test'
 import { setup } from '../shared-e2e'
 
-test.describe('ui', () => {
+test.describe('mobile ui', () => {
 	test.beforeEach(setup)
-
-	test('mobile style panel opens and closes when tapped', async ({ isMobile }) => {
+	test('mobile style panel opens and closes when tapped', async ({ isMobile, page }) => {
```

Comment:
We might not needto repeat mobile here as the parent descript already mentions it?